node/src/cli.rs

```diff
@@ -238,6 +238,7 @@ impl FromStr for Sealing {
 pub enum EthApi {
 	Txpool,
 	Debug,
+	Trace,
 }
```

Comment:
Design question. Is this making the node operator choose which one of these endpoints to run in the node? I could imagine someone wanting to use txpool and trace together. Is this not useful in practice?

Reply:
Ahh, nevermind. I see you can pass a vec of them.